app/src/main/java/org/dicio/dicio_android/eval/SkillEvaluator.java

```diff
@@ -353,14 +372,40 @@ private void evaluateMatchingSkill(final String input) {
             final List<String> inputWords = WordExtractor.extractWords(input);
             final List<String> normalizedWordKeys = WordExtractor.normalizeWords(inputWords);
             final Skill skill = skillRanker.getBest(input, inputWords, normalizedWordKeys);
-
+            if(skill==null) {
+                Log.w("SKILLS", "No matching Skill found");
+                return null;
+            }
```

Comment:
This should not be needed, since `getBest` always returns a non-null skill (if no valid skill was found the fallback skill is returned).